articles/azure-functions/durable/durable-functions-concepts.md

```diff
@@ -34,6 +34,25 @@ In the function chaining pattern, a sequence of functions executes in a specific
 
 You can use Durable Functions to implement the function chaining pattern concisely as shown in the following example:
 
+#### Precompiled C#
+
+```csharp
+public static async Task<object> Run([OrchestrationTrigger] DurableOrchestrationContext context)
```

Comment:
I see my previous comment was marked as resolved, but there's still no `[FunctionName(...)]` attribute. Is there another commit that needs to be pushed to this PR?